tests/e2e/jest.config.json

```diff
@@ -3,6 +3,16 @@
   "testEnvironment": "node",
   "setupFilesAfterEnv": ["<rootDir>/_setupLogging.ts"],
   "testTimeout": 180000,
+  "extensionsToTreatAsEsm": [".ts"],
+  "transform": {
```

Comment:
Remove the `"preset": "ts-jest"` if we're using transform: https://kulshekhar.github.io/ts-jest/docs/guides/esm-support/